### PR TITLE
fix pam configuration on SLES15SP6

### PR DESCRIPTION
**Pull request by mbussolotto**
## What does this PR change?

fix pam configuration on SLES15SP6

## GUI diff

No difference.

Before:

After:

- [ ] **DONE**

## Documentation
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- No documentation needed: only internal and user invisible changes
- Documentation issue was created: [Link for SUSE Manager contributors](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31), [Link for community contributors](https://github.com/uyuni-project/uyuni-docs/issues/new).
- API documentation added: please review the Wiki page [Writing Documentation for the API](https://github.com/uyuni-project/uyuni/wiki/Writing-documentation-for-the-API) if you have any changes to API documentation.
- (OPTIONAL) [Documentation PR](https://github.com/uyuni-project/uyuni-docs/pulls)

- [ ] **DONE**

## Test coverage
- No tests: **add explanation**
- No tests: already covered
- Unit tests were added
- Cucumber tests were added

- [ ] **DONE**

## Links

Issue(s): #
Port(s): # **add downstream PR(s), if any**

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: containers/server-image/Dockerfile [hunk 1]
```diff
@@ -60,9 +62,7 @@ RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-li
     ipmitool \
     sssd \
     sssd-dbus \
-    sssd-ldap \
-    nss-pam-ldapd \
-    authselect
```
Comment (cbosdo) [inline containers/server-image/Dockerfile:65]: couldn't you add those two packages to the EXTRA_PACKAGES argument as default value?

**Review by cbosdo — APPROVED** [review]

**Comment by mbussolotto** [comment]
Testing in my OBS project before merging it

File: containers/server-image/Dockerfile [hunk 2]
```diff
@@ -7,12 +7,14 @@ FROM $INIT_IMAGE
 ARG PRODUCT_PATTERN_PREFIX="patterns-uyuni"
 
 # Extra packages can be added via project configuration
-ARG EXTRAPACKAGES
+ARG EXTRAPACKAGES=shim nss-pam-ldapd authselect
```
Comment (cbosdo) [inline containers/server-image/Dockerfile:10]: don't add `shim` here as this is depending on the architecture we build on

**Review by cbosdo — CHANGES_REQUESTED** [review]
Don't forget to merge your commits in one

File: containers/server-image/Dockerfile [hunk 3]
```diff
@@ -7,12 +7,14 @@ FROM $INIT_IMAGE
 ARG PRODUCT_PATTERN_PREFIX="patterns-uyuni"
 
 # Extra packages can be added via project configuration
-ARG EXTRAPACKAGES
+ARG EXTRAPACKAGES=shim%20nss-pam-ldapd
```
Comment (cbosdo) [inline containers/server-image/Dockerfile:10]:
```suggestion
ARG EXTRAPACKAGES=nss-pam-ldapd
```
`shim` should not be in the defaults: it is only for some architectures

**Review by cbosdo — APPROVED** [review]

File: containers/server-image/Dockerfile [hunk 4]
```diff
@@ -61,8 +61,10 @@ RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-li
     sssd \
     sssd-dbus \
     sssd-ldap \
-    nss-pam-ldapd \
-    authselect
+    sssd-ad \
```
Comment (deneb-alpha) [inline containers/server-image/Dockerfile:64]: I see it available https://smelt.suse.de/maintained/?q=sssd-ad&with_debug=1

File: containers/server-image/Dockerfile [hunk 5]
```diff
@@ -61,8 +61,10 @@ RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-li
...
+    sssd-ipa \
```
Comment (deneb-alpha) [inline containers/server-image/Dockerfile:65]: this too https://smelt.suse.de/maintained/?q=sssd-ipa&with_debug=1

File: containers/server-image/Dockerfile [hunk 6]
```diff
@@ -61,8 +61,10 @@ RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-li
...
+    sssd-krb5 \
```
Comment (deneb-alpha) [inline containers/server-image/Dockerfile:66]: same https://smelt.suse.de/maintained/?q=sssd-krb5&with_debug=1

File: containers/server-image/Dockerfile [hunk 7]
```diff
@@ -61,8 +61,10 @@ RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-li
...
+    sssd-tools
```
Comment (deneb-alpha) [inline containers/server-image/Dockerfile:67]: same https://smelt.suse.de/maintained/?q=sssd-tools&with_debug=1

File: containers/server-image/server-image.changes.mbussolotto.pam_package [hunk 8]
```diff
@@ -0,0 +1 @@
+- fix PAM authentication using bci image
```
Comment (deneb-alpha) [inline containers/server-image/server-image.changes.mbussolotto.pam_package:1]:
```suggestion
- Fix PAM authentication using BCI image
```

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
the subpackages of https://smelt.suse.de/maintained/?q=sssd&with_debug=1 are all there, not everything is already in Micro 5.5 but this isn't a problem for us to build the  image. I think we are good to  go  in this way.

I left a suggestion for the .changes

**Review by cbosdo — APPROVED** [review]

**Review by deneb-alpha — APPROVED** [review]
LGTM,thanks

**Pull request merged**